CLAUDE.md

```
@@ -1,44 +1,175 @@
-- Leave `## [Unreleased]` empty at the top
-- When adding entries to an existing `[Unreleased]` section, append new items at the **end** of the relevant category (e.g. `### Changed`), not at the top
+## Patterns & Conventions
 
-Example structure:
-```markdown
-# Changelog (Android)
+### Feature Module Pattern
 
-## [Unreleased]
+Each feature follows this structure:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)